app/helpers/polls_helper.rb

```diff
@@ -0,0 +1,47 @@
+module PollsHelper
+    MAX = 26**4-1; # the maximum number that can be hashed to
```

Comment:
This is the old hashing algorithm. The new one was pushed with the assignment bad word removal ticket